styles.css

```diff
@@ -0,0 +1,99 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-family: 'Audiowide', cursive;
+}
+
+.main-wrapper {
+    height: 100vh;
+    width: 100vw;
+    background: #5433FF;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    align-items: center;
+    justify-content: center;
+    display: flex;
+}
+
+.content-wrapper {
+    background-color: #ffffff;
```

Comment:
we can use shorter version `#fff`. 
Please use in all colors small or big letters (you can choose whatever you want)